### DISCUSSION TITLE
可以修改悬浮图标位置吗

### DISCUSSION THREAD

**Discussion by gqyanann** [the1812/Bilibili-Evolved#4479] [功能建议 / Ideas]
现在的位置每次靠近都会触发我的应该常用软件的悬浮窗，很麻烦

**Comment by Geeyun-JY3** [comment]
设置面板的`通用设置`里可以设置放左边还是右边，[`侧栏垂直偏移`](https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features/features.md#%E4%BE%A7%E6%A0%8F%E5%9E%82%E7%9B%B4%E5%81%8F%E7%A7%BB)可以设置垂直偏移量。
